src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { useAccount } from 'wagmi';
+import { DataList, Tag } from '../../../../../core';
+import { addressUtils } from '../../../../utils/addressUtils';
+import { ApprovalThresholdResult } from '../approvalThresholdResult';
+import { MajorityVotingResult } from '../majorityVotingResult';
+import { ProposalDataListItemStatus } from '../proposalDataListItemStatus';
+import {
+    type IApprovalThresholdResult,
+    type IMajorityVotingResult,
+    type IProposalDataListItemStructureProps,
+} from './proposalDataListItemStructure.api';
+
+/**
+ * `ProposalDataListItemStructure` module component
+ */
+export const ProposalDataListItemStructure: React.FC<IProposalDataListItemStructureProps> = (props) => {
+    const {
+        wagmiConfig: config,
+        chainId,
+        type,
+        result,
+        date,
+        protocolUpdate,
+        publisher,
+        publisherProfileLink,
+        status,
+        summary,
+        title,
+        voted,
+        ...otherProps
+    } = props;
+
+    const { address: connectedAddress, isConnected } = useAccount({ config });
+
+    const ongoing = status === 'active' || status === 'challenged' || status === 'vetoed';
+
+    const publisherIsConnected = isConnected && connectedAddress?.toLowerCase() === publisher.address?.toLowerCase();
+    const publisherLabel = publisherIsConnected
+        ? 'You'
+        : publisher.name ?? addressUtils.shortenAddress(publisher.address as string);
+
+    return (
+        <DataList.Item className="space-y-4" {...otherProps}>
+            <ProposalDataListItemStatus date={date} status={status} voted={voted} />
+            <div className="space-y-1">
```

Comment:
I started noticing these `space-*` classes on other components as well, I would suggest to always prefer `gap-*` to set the spacing between the components

Reply:
What's the reasoning behind that? Adding gap means we have to tranform this into either a grid or a flexbox.

Reply:
In general I think that the gap property better represent the spacing idea between elements and it actually applies this in flex/grid containers. `space-*` instead applies margins on elements making sure that such margins are not applied on the first/last elements, sounds a bit more convoluted as idea

Reply:
Prior to flexbox, that was _the only_ way to do it. I want you to know that I looked up (and couldn't find) an "are you serious" emoji 😅

Reply:
We should create such emoji :D